lib/server.js

```diff
@@ -305,6 +305,13 @@ function create(log, error, config, routes, db, translator) {
     }
   )
 
+  server.ext('onPreHandler', (request, reply) => {
```

Comment:
> Is the intent to enable `features` for other routes?

Yep, I tried to point this out in my comment:

> I've tried to do this somewhat generically so that the same pattern can easily be applied to other endpoints as necessary.

If you think I'm guilty of over-engineering then I'm happy to make the change, although fwiw pushing stuff on to `request.app` is a pattern we already follow elsewhere. See, for instance, the code for locales just above this in the `onPreAuth` hook.